src/wx/wayland.h

```diff
@@ -1 +1,15 @@
-bool IsItWayland();
+#ifndef VBAM_WAYLAND_H
+#define VBAM_WAYLAND_H
+
+#include <wx/version.h>
+#include <wx/window.h>
+#include <wx/glcanvas.h>
+
+bool IsWayland();
+
+// Temporary hack to backport 800d6ed69b from wxWidgets until 3.2.2 is released.
+#if defined(__WXGTK__) && wxCHECK_VERSION(3, 2, 0) && !wxCHECK_VERSION(3, 2, 2)
+void MoveWaylandSubsurface(wxGLCanvas* win);
```

Comment:
I'd suggest leaving the definition in the header and making it a no-op on non-wayland platforms to fix the build issue.